guide/14-deep-learning/dl-on-time-series-data.ipynb

```diff
@@ -0,0 +1,301 @@
+{
```

Comment:
<p>Once the explanatory variables and explanatory rasters are defined, we can prepare our TabularDataObject by using the<code>prepare_tabulardata()</code> function. The following are some of the key arguments for this function: </p><ul><li><code>input_features</code>: Takes data in the form of a feature layer, spatially enabled DataFrame, or non spatial dataframe. Feature layers can be of point or polygon geometry.</li><li><br></li><li><code>variable_predict</code>: Field name from the&nbsp;<code>input_features</code>&nbsp;layer or&nbsp;<code>explanatory_rasters</code>. It automatically deduces fields as continuous or categorical by using a simple logic. If the field type is float and has more than 20 unique values, the field is considered continuous. Otherwise, the field will be considered categorical.</li><li><br></li><li><code>date_field</code>: Field name from the&nbsp;<code>input_features</code>&nbsp;layer. If specified, the field will be split into year, month, week, day, dayofweek, dayofyear, is_month_end, is_month_start, is_quarter_end, is_quarter_start, is_year_end, is_year_start, hour, minute, second, elapsed. If specified here, there is no need to specify as part of&nbsp;<code>explanatory_variables</code>.</li><li><br></li><li><code>explanatory_variables</code>&nbsp;: List of the field names containing the explanatory variables from the&nbsp;<code>input_layer</code>, where tuples of size 2 have to be passed for categorical variables, as discussed above. For univariate cases, these are not required.</li><li><br></li><li><code>explanatory_rasters</code>&nbsp;: List of the Raster objects containing the explanatory variables, where tuples of size 2 have to be passed for categorical variables, as discussed above.</li><li><br></li><li><code>distance_features</code>: Distance features are used to automatically create explanatory variables by calculating a distance from the provided features to the input_features. Distances will be calculated from each of the input explanatory distance features to the nearest input training features. If the input&nbsp;<code>distance_features</code>&nbsp;are polygons or lines, the distance attributes are calculated as the distance between the closest segments of the pair of features. However, distances are calculated differently for polygons and lines. See how&nbsp;<a href="https://pro.arcgis.com/en/pro-app/tool-reference/analysis/how-near-analysis-works.htm" target="_blank">proximity tools calculate distance</a>&nbsp;for details.</li><li><br></li><li><code>index_field</code>&nbsp;: Name of the field that contains the indexes for the data. This is useful when visualizing the time series data. It is usually a timestamp or date time field.</li><li><br></li></ul><p>Once the explanatory variables or the explanatory rasters list is ready, it can be used as input for the <code>prepare_tabulardata()</code> function. The function takes the feature layer or a spatial DataFrame containing the dataset, along with the y-variable defined as the variable_predict, and prepares it for fitting the <code>TimeSeriesNetwork</code> model as follows:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>